uui-core/src/hooks/usePrevious.ts

```diff
@@ -1,8 +1,8 @@
 import { useEffect, useRef } from 'react';
 import isEqual from 'lodash.isequal';
 
-export function usePrevious<T>(value: T) {
-    const previousValueRef = useRef(value);
+export function usePrevious<T>(value: T): T | null {
```

Comment:
We currently export these hooks. We need to either remove them from exports for now, or document them. 

It's also not clear from their names what's the difference between them. Probably, it's better to have a single hook with a prop like 'compare'.

Reply:
Removed from exports, merged with useSimplePrevious, and added options with compare: boolean.